### DISCUSSION TITLE
Documents for `Engine` class?

### DISCUSSION THREAD

**Discussion by pjyi2147** [sgl-project/sglang#1613] [Q&A]
I saw that separation of `Engine` and server is now implemented, but I do not see any documents in the `readthedocs` website. Will be the usages/api endpoints updated soon?

**Comment by merrymercy** [comment]
Yes! For now, please see this example https://github.com/sgl-project/sglang/blob/main/examples/runtime/srt_engine.py and PR https://github.com/sgl-project/sglang/pull/1614.

**Comment by ByronHsu** [comment]
https://sglang.readthedocs.io/en/latest/backend.html#engine-without-http-server
The doc is now live
